eng/scripts/Create-ApiReview.ps1

```diff
@@ -0,0 +1,185 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [array] $ArtifactList,
```

Comment:
Given these are different than what is currently in common I think you are going to have some difficulties moving this into common, unless you plan to move everyone away from common temporarily and then move them back to common afterwards.

Reply:
I prefer to get al languages using separate path first and then merge them to use single common scripts.